vowpalwabbit/core/src/reductions/eigen_memory_tree.cc

```diff
@@ -710,34 +869,14 @@ size_t write_model_field(
 size_t read_model_field(io_buf& io, reductions::eigen_memory_tree::emt_tree& tree)
 {
   size_t bytes = 0;
-
-  bytes += read_model_field(io, tree.leaf_split);
```

Comment:
Since emt has been released we need to be careful about changed to save/load.

If you want to change this save/load code then please add a check  in your save load function which throws an error if a model is loaded which is of a version before these changes were made. You can find examples of this in other reductions

Reply:
Oh, that makes sense. The save changes aren't needed. My vote would be that maintaining two separate save versions probably isn't worth it in this case. Unless you feel differently, I've simply put back the old save logic without adding the new flag.

Reply:
Yeah i agree, leaving as is is the least disruption